Libraries/CMSIS/Device/Maxim/MAX32657/Include/max32657.h

```diff
@@ -601,52 +615,70 @@ We may want to handle GET_IRQ better...
 #define MXC_I3C_S ((mxc_i2c_regs_t *)MXC_BASE_I3C_S)
 
 #if IS_SECURE_ENVIRONMENT
+#define MXC_BASE_I3C MXC_BASE_I3C_S
 #define MXC_I3C MXC_I3C_S
 #else
+#define MXC_BASE_I3C MXC_BASE_I3C_NS
 #define MXC_I3C MXC_I3C_NS
 #endif
 
 /******************************************************************************/
 /*                                                                        DMA */
 #define MXC_DMA_CHANNELS (4)
-#define MXC_DMA_INSTANCES (1)
-// ^ Note: We have 2 DMA instances in hardware, but they are secure vs non-secure
-// instances.  Therefore we treat the part as if there is only 1.
+#define MXC_DMA_INSTANCES (2)
```

Comment:
If you try to build with this you'll see an issue at the bottom-level DMA driver file with the handler definitions.  Also RevA depends on `MXC_DMA`

Reply:
Yes, that is why I added a WIP for this PR before committing to this.